macros/base/end_print.cfg

```diff
@@ -6,7 +6,6 @@ gcode:
     {% set safe_extruder_temp = printer["gcode_macro _USER_VARIABLES"].safe_extruder_temp|float %}
     {% set light_intensity_end_print = printer["gcode_macro _USER_VARIABLES"].light_intensity_end_print %}
     {% set klippain_mmu_enabled = printer["gcode_macro _USER_VARIABLES"].klippain_mmu_enabled %}
```

Comment:
did you try like that?
```suggestion
    {% set klippain_mmu_enabled = printer["gcode_macro _USER_VARIABLES"].klippain_mmu_enabled %}
    {% set CHECK_GATES = params.CHECK_GATES|default(printer['gcode_macro _USER_VARIABLES'].mmu_check_gates_on_start_print,0) %} # Check if MMU gates are availables
```

Reply:
that doesn't make sense. you added source code that i removed because it throws an error. (line 8)
regarding line 9: why haven't you added it in the first place like this? ;)

Reply:
default on non existing variable errors out with dict error unknown. so that idea is not working

Reply:
I make a mistake between start_print and end_print...
```suggestion
    {% set mmu_unload = params.MMU_UNLOAD_AT_END|default(printer["gcode_macro _USER_VARIABLES"].mmu_unload_on_end_print,0) %}
```

Reply:
```suggestion
    {% set klippain_mmu_enabled = printer["gcode_macro _USER_VARIABLES"].klippain_mmu_enabled %}
    {% set mmu_unload = params.MMU_UNLOAD_AT_END|default(printer["gcode_macro _USER_VARIABLES"].mmu_unload_on_end_print,0) %}
```